addons/hr_recruitment/views/hr_applicant_views.xml

```diff
@@ -323,6 +323,7 @@
                 <field name="user_id"/>
                 <field name="active"/>
                 <field name="application_status" />
+                <field name="company_id" invisible="1"/>
```

Comment:
You just have to add a comment to explain why this field should remain in the view